### PR TITLE
[12.0][ADD] maintenance_equipment_always_fold

### PR DESCRIPTION
**Pull request by lmiguens-solvos**
Equipment categories will always be folded if I indicate it with an always_fold Boolean; if not, let him behave as before

### CODE REVIEW

File: maintenance_equipment_always_fold/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Maintenance Equipment Categories always_fold is indicate it",
+    "summary": "Equipment categories will always be folded if I indicate it "
+               "with an always_fold Boolean; if not, let him behave as before",
+    "author": "Odoo Community Association (OCA), Solvos",
+    "license": "AGPL-3",
+    "version": "12.0.1.2.0",
```
Comment (dalonsod) [inline maintenance_equipment_always_fold/__manifest__.py]: Wrong version

File: maintenance_equipment_always_fold/models/maintenance.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models, _
+
+
+class MaintenanceEquipmentCategory(models.Model):
+    _inherit = 'maintenance.equipment.category'
+
+    @api.one
```
Comment (dalonsod) [inline maintenance_equipment_always_fold/models/maintenance.py]: Deprecated, please remove it

File: maintenance_equipment_always_fold/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "category": "Equipments",
+    "website": "https://github.com/OCA/maintenance",
+    "depends": [
+        "maintenance",
+    ],
+    "data": [
+        'views/maintenance_views.xml',
+    ],
+    "demo": [
```
Comment (dalonsod) [inline maintenance_equipment_always_fold/__manifest__.py:19]: Maybe some demo data could be added

**Review by dalonsod — CHANGES_REQUESTED** [review]
See travis and codecov

File: maintenance_equipment_always_fold/__manifest__.py (hunk 3 above)
Comment (lmiguens-solvos) [reply maintenance_equipment_always_fold/__manifest__.py:19]: Done

File: maintenance_equipment_always_fold/models/maintenance.py (hunk 2 above)
Comment (lmiguens-solvos) [reply maintenance_equipment_always_fold/models/maintenance.py]: Done

File: maintenance_equipment_always_fold/__manifest__.py (hunk 1 above)
Comment (lmiguens-solvos) [reply maintenance_equipment_always_fold/__manifest__.py]: New version put

**Comment by lmiguens-solvos** [comment]
@etobella, @dalonsod, can you review this PR? Thanks.

**Review by dalonsod — APPROVED** [review]
Tested on runbot 👍 
Not blocking, `always_fold` field could be added to equipment categories tree view.

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Comment by dalonsod** [comment]
hello @etobella is this PR ready to be merged? Thanks!!

**Pull request merged**